build.gradle.kts

```diff
@@ -1,32 +1,30 @@
 plugins {
-    id("org.jetbrains.intellij") version "1.9.0"
+    id("org.jetbrains.intellij") version "1.11.0"
     java
-    kotlin("jvm") version "1.6.20"
+    kotlin("jvm") version "1.7.22"
 }
 
 group "com.asyncapi.plugin.idea"
-version = "1.7.1+idea2021"
+version = "1.7.1+idea2022"
```

Comment:
Might I suggest this be `1.8.1+idea2022`?

If so, this PR should be against _that_ release branch

Reply:
isn't this the 1.8.0 release branch?